scripts/install_environment.sh

```diff
@@ -43,6 +43,21 @@ function install_libvirt() {
         swtpm \
         swtpm-tools
 
+    # Just testing, do not merge.
+    sudo curl https://github.com/nmagnezi/swtpm/raw/PR-645/bin/swtpm > /usr/bin/swtpm
```

Comment:
maybe you'll need to do this, but let's see:
```suggestion
    sudo curl https://github.com/nmagnezi/swtpm/raw/PR-645/bin/swtpm -o /usr/bin/swtpm
```

Reply:
Tested this locally and it worked like that.  I did that to override the existing files deployed from the RPMs